src/select/ValueGroup.js

```diff
@@ -4,12 +4,6 @@ import PropTypes from 'prop-types'
 import { getSelectValue } from './utils'
 
 class ValueGroup extends Component {
-  onMouseDown = (event) => {
-    if (this.props.onMouseDown) {
```

Comment:
Why remove this, was it not used?

Reply:
Nope, after upgrading eslint, it flagged it as not used. I couldn't find any usages of it 🤷🏽